core/api/src/app/users/list-sessions.ts

```diff
@@ -0,0 +1,10 @@
+import { consentList } from "@/services/hydra"
+import { listSessions as listSessionsService } from "@/services/kratos"
+
+export const listMobileSessions = async (userId: UserId) => {
```

Comment:
please include return type and change param to `{ userId }: {userId: UserId }` in this way params is passed correctly for tracing otherwise will be code.function.params.0

Reply:
adding that to ACE... so we can understand where it's necessary to have an object as opposed to an simple argument when there is a single scalar passed to the functions